verticaltabs.js

```diff
@@ -313,6 +314,21 @@ VerticalTabs.prototype = {
       }
     }
 
+    //if new tab button is not in toolbar, find it and insert it.
+    if (!toolbar.querySelector('#new-tab-button')) {
+      //save position of button for restoring later
+      let NewTabButton = document.getElementById('new-tab-button') || CustomizableUI.getWidget('new-tab-button').forWindow(this.window).node;
```

Comment:
Why don't we always use `CustomizableUI.getWidget('new-tab-button').forWindow(this.window).node`, and skip the `||`?